hwconf/hw_Cheap_FOCer_2.h

```diff
@@ -20,7 +20,7 @@
 #ifndef HW_Cheap_FOCer_2
 #define HW_Cheap_FOCer_2
 
-#define HW_NAME					"Cheap FOCer 2"
+#define HW_NAME					"Cheap_FOCer_2"
```

Comment:
It'd be awesome to add `_v10` here or something so as not to mix it up with the popular version 0.9

Reply:
0.9 was beta, this is the first release and will be the standard hardware going forward. It might be a little confusing for some people now, but this is the cleanest path forward.

FWIW this firmware wont overwrite firmware on a beta unit using the tool. You'd have to build it from scratch and flash with an st-linker. Additionally there is a migration guide in the main repo for making old hardware compatible with this firmware.

Lastly flashing this firmware on a beta unit just lowers your current values by half but works fine, not a huge risk or danger.